doc/contribute/guidelines.rst

```diff
@@ -532,9 +532,18 @@ results you have to create an account yourself.  From the Zephyr project page,
 you may select "Add me to project" to be added to the project. New members must
 be approved by an admin.
 
-Coverity scans the Zephyr codebase weekly. GitHub issues are automatically
-created for any problems found and assigned to the maintainers of the affected
-areas.
+Static analysis of the Zephyr codebase is conducted on a bi-weekly. GitHub
```

Comment:
previous request for change was only partially incorporated leading to a missing word:

```suggestion
Static analysis of the Zephyr codebase is conducted on a bi-weekly basis. GitHub
```